pkg/git/providers/git/git.go

```diff
@@ -0,0 +1,71 @@
+package git
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+)
+
+const (
+	GitPasswordTokenEnv       = "GIT_PASSWORD"
+	EksaGitPasswordTokenEnv   = "EKSA_GIT_PASSWORD"
+	GitPrivateKeyTokenEnv     = "GIT_PRIVATE_KEY"
```

Comment:
do we need `GIT_PRIVATE_KEY`?

I know we need `GIT_PASSWORD` as the env var we are passing to `flux bootstrap git`, but I think we can just read `EKSA_GIT_PRIVATE_KEY` straight from env and not worry about having `GIT_PRIVATE_KEY` as an intermediate env var, since we're setting the file path as a string in the command execution

Reply:
that's true, we don't need this var